assets/src/settings-page/index.js

```diff
@@ -237,13 +253,11 @@ function Root() {
 			</form>
 			<UnsavedChangesWarning excludeUserContext={ true } />
 			{ error && <ErrorNotice errorMessage={ error.message || __( 'An error occurred. You might be offline or logged out.', 'amp' ) } /> }
-			{ saved && (
-				<AMPNotice className={ `amp-save-success-notice` } type={ NOTICE_TYPE_SUCCESS }>
-					<p>
-						{ __( 'Settings saved', 'amp' ) }
-					</p>
-				</AMPNotice>
-			) }
+			<AMPNotice className={ `amp-save-success-notice ${ savedNoticeClass }` } type={ NOTICE_TYPE_SUCCESS }>
```

Comment:
Given that this is no longer conditionally added to the page when an update happens, does this mean it won't be accessible for screen readers in the same way?

Reply:
Good point. Let me hide an `aria-hidden`

Reply:
Done in 5adfee2aa

Reply:
OK, so now it is hidden from screen readers… but eventually they should also announce that changes were made successfully. I believe at the moment screen readers would not give any indication that a save was successful.

Probably not something we need to tackle for 2.0, but something we should keep in mind as we want the plugin to be accessible to as many users as possible.

Reply:
Oh, the `aria-hidden` is conditionally shown based on whether it is visible. Got it.

Reply:
This will have `aria-hidden="false"` when the component is visible, but you're right that the flyout experience might be different for screen-reader users. I'll look at how Gutenberg does the snackbar component (for 2.1).